spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestRemoveOrphanFilesAction.java

```diff
@@ -150,10 +150,20 @@ public void testDryRun() throws IOException, InterruptedException {
     Assert.assertEquals("Action should find 1 file", invalidFiles, result2.orphanFileLocations());
     Assert.assertTrue("Invalid file should be present", fs.exists(new Path(invalidFiles.get(0))));
 
+    String actualFilesTableName = "actualFilesTable";
+    spark.createDataset(allFiles, Encoders.STRING()).toDF("file_path").createOrReplaceTempView(actualFilesTableName);
```

Comment:
I think that this test case should run _after_ the actual orphan file delete. Otherwise, listing produces the same result and we're not actually verifying that we get a different result from the new option.